Apps/PowerShell/ActiveDirectory/Apps.PowerShell.ActiveDirectory.psd1

```diff
@@ -51,7 +51,7 @@
     # ProcessorArchitecture = ''
 
     # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules = @('ActiveDirectory.Scripts')
+    # RequiredModules = @('ActiveDirectory.Scripts')
```

Comment:
this fixed the build....

Reply:
@adamdriscoll, did you really want to require the module like this?

Reply:
I did. The idea being that installing the AD app in PSU would install the AD scripts module if it wasn't already installed. I think the problem is that the build script is just publishing modules in order without taking into account the dependencies so it fails.

Reply:
you can check out my branch and work from there instead of smashing stuff into main ;) if you like